healthcare/dataset_iam_set.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START healthcare_dataset_set_iam_policy]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	healthcare "google.golang.org/api/healthcare/v1beta1"
+)
+
+// setDatasetIAMPolicy sets an IAM policy for the dataset.
+func setDatasetIAMPolicy(w io.Writer, projectID, location, datasetID string) error {
```

Comment:
we should probably talk to Benedict about this sample. I don't think it's very useful - it seems pretty rare you'd set a policy outright, rather than augmenting an existing one.

Reply:
Done. Changed to append (because it doesn't add much complexity, but matches the shell sample Benedict pointed out).